dev-project/pipelinewise-config/tap_mongodb_to_pg.yaml

```diff
@@ -3,7 +3,7 @@
 # ------------------------------------------------------------------------------
 # General Properties
 # ------------------------------------------------------------------------------
-id: "tap_mongo"
+id: "tap_mongo_to_pg"
```

Comment:
can we make this `mongo_to_pg` to keep it in sync with https://github.com/transferwise/pipelinewise/pull/434/commits/c88330af5082a89b11a83ef0ab6984dbadd86afb ?

Reply:
Fixed in commit https://github.com/transferwise/pipelinewise/pull/434/commits/0dd583ffa908b5f83a67a13d3baa1c69df949952